pkg/server/plugin/datastore/sql/sql.go

```diff
@@ -679,24 +676,29 @@ func applyBundleMask(model *Bundle, newBundle *common.Bundle, inputMask *common.
 		return nil, nil, err
 	}
 
-	if inputMask.RefreshHint {
+	m := inputMask
+	if m == nil {
+		m = protoutil.AllTrueCommonBundleMask
```

Comment:
nit: i might suggest just setting inputMask to the all-true mask instead of creating another variable. It reduces the chance that a future refactoring accidentally uses `inputMask` instead of `m`:
```suggestion
	if inputMask == nil {
		inputMask = protoutil.AllTrueCommonBundleMask
```